spec/features/space_spec.rb

```diff
@@ -1,8 +1,17 @@
 feature 'creating a new space' do
-  scenario 'user creates a new space' do
-    visit '/space/new'
-    fill_in :name, with: "newspace"
-    click_button("create space")
-    expect(Space.first.name).to eq "newspace"
+  scenario 'user creates a new space name' do
+   register_space
+    expect(Space.first.name).to eq"newspace"
```

Comment:
check spacing
- space before/after scenario? they're present later so pick one for consistency
- expect and register_space are not inline